lookout/style/format/benchmarks/top_repos_quality.py

```diff
@@ -211,26 +210,18 @@ def wrapped_capture_quality_report(*args, **kwargs):
 
 def calc_weighted_avg(arr: Sequence[Sequence], col: int, weight_col: int = 5) -> float:
     """Calculate average value in `col` weighted by column `weight_col`."""
-    numerator, denominator = 0, 0
-    # TODO: switch to numpy arrays
-    for row in arr:
-        numerator += row[col] * row[weight_col]
-        denominator += row[weight_col]
+    arr = numpy.array(arr)
```

Comment:
`arr` here can contain ints, floats as well as strings. Does this code work well in this case?

Reply:
`.astype(float)` added to be sure that it works.

Reply:
ok, I hope it is fine now.